### PR TITLE
HODS-388 | Removed sandbox implementation

### PR DESCRIPTION
**Pull request by matthewharris1**

### CODE REVIEW

File: app/uk/gov/hmrc/individualsbenefitsandcreditsapi/controllers/RootController.scala [hunk 1]
```diff
@@ -62,38 +63,5 @@ abstract class RootController @Inject()(
...
-    auditHelper: AuditHelper,
-    sandboxTaxCreditsService: SandboxTaxCreditsService
-)(implicit override val ec: ExecutionContext)
-    extends RootController(cc,
-                           scopeService,
-                           scopesHelper,
-                           auditHelper,
-                           sandboxTaxCreditsService) {
-  override val environment = Environment.SANDBOX
+  val environment = Environment.PRODUCTION
```
Comment (owen-gilfellon) [inline app/uk/gov/hmrc/individualsbenefitsandcreditsapi/controllers/RootController.scala]: This isn't used anymore, can be deleted

File: app/uk/gov/hmrc/individualsbenefitsandcreditsapi/controllers/WorkingTaxCreditController.scala [hunk 2]
```diff
@@ -63,27 +64,5 @@ abstract class WorkingTaxCreditController @Inject()(
...
-@Singleton
-class SandboxWorkingTaxCreditController @Inject()(
-    val authConnector: AuthConnector,
-    cc: ControllerComponents,
-    scopeService: ScopesService,
-    auditHelper:AuditHelper,
-    taxCreditsService: SandboxTaxCreditsService
-) extends WorkingTaxCreditController(cc, scopeService, auditHelper, taxCreditsService) {
-  override val environment = Environment.SANDBOX
+  val environment = Environment.PRODUCTION
```
Comment (owen-gilfellon) [inline app/uk/gov/hmrc/individualsbenefitsandcreditsapi/controllers/WorkingTaxCreditController.scala]: Not used anymore, can be removed

File: app/uk/gov/hmrc/individualsbenefitsandcreditsapi/controllers/ChildTaxCreditController.scala [hunk 3]
```diff
@@ -66,28 +67,5 @@ abstract class ChildTaxCreditController @Inject()(
...
-class SandboxChildTaxCreditController @Inject()(
-    val authConnector: AuthConnector,
-    cc: ControllerComponents,
-    scopeService: ScopesService,
-    auditHelper:AuditHelper,
-    sandboxTaxCreditsService: SandboxTaxCreditsService
-)(implicit override val ec: ExecutionContext)
-    extends ChildTaxCreditController(cc, scopeService, auditHelper, sandboxTaxCreditsService) {
-  override val environment = Environment.SANDBOX
+  val environment = Environment.PRODUCTION
```
Comment (owen-gilfellon) [inline app/uk/gov/hmrc/individualsbenefitsandcreditsapi/controllers/ChildTaxCreditController.scala]: Not used anymore, can be removed

**Review by owen-gilfellon — CHANGES_REQUESTED** [review]

**Review by owen-gilfellon — APPROVED** [review]

**Review by ajchilts — APPROVED** [review]
Please address @owen-opencast review comments then we can run regression pack and merge. Nice job. 👍

**Review by ajchilts — APPROVED** [review]

**Pull request merged**